### ISSUE TITLE
idx

### ISSUE THREAD

**Issue by gBlaku** [elebumm/RedditVideoMakerBot#380]
Any idea why this happens? Many times.
![err](https://user-images.githubusercontent.com/48861657/172473667-c82bf6cb-8083-4d1a-8ecb-a782e60cabd4.png)

**Comment by gBlaku** [comment]
Yes I added one extra line at the end. My goal is to create videos nonstop. The error only occurs sometimes.. but it stops the whole process.
![image](https://user-images.githubusercontent.com/48861657/172482483-b0455ff0-13ec-432b-af0b-34138d3bd208.png)

**Comment by naufalfitra** [comment]
i write the exact same code, but add one space by pressing enter before the "os code"
it works well
